docs/components/tabs.md

```diff
@@ -1,180 +1,204 @@
 # Tabs
 
-**KTabs** - A mindblowing tabs component
+KTabs are horizontal controls that allow users to switch between multiple views within one page.
 
 <KTabs :tabs="tabs">
-  <template v-slot:tab1>
+  <template #tab1>
```

Comment:
issue: the third tab in the example here is disabled, but is not shown as disabled in the example. I'm not sure we need `disabled` here in the first example since we don't mention it regardless.

Reply:
...and there's actually a disabled tab in **most** of the examples -- this is unexpected. Can it just be a normal tab please and use a `disabled` one under the corresponding docs section?

Reply:
Adjusted the disabled item title to mention that it's disabled. I think it's useful to have it a few examples.

Reply:
...but it's in ALL the examples?